src/main/java/bowling/score/Score.java

```diff
@@ -18,34 +23,53 @@ private Score(int value, int bonusCount) {
 		this.bonusCount = bonusCount;
 	}
 
-	public static Score of(int value, int bonusCount) {
-		return new Score(value, bonusCount);
+	public static Score unavailable() {
+		return new Score(UNAVAILABLE_SCORE, UNAVAILABLE_COUNT);
+	}
+
+	public static Score remain(Score first) {
+		return Score.of(first.value, MIN_BONUS_COUNT);
 	}
 
 	public static Score open(Score first, Score second) {
-		return new Score(first.value + second.value, OPEN_BONUS_COUNT);
+		return Score.of(first.value + second.value, MIN_BONUS_COUNT);
 	}
 
-	public static Score spare(Score maxScore) {
-		return new Score(maxScore.value, SPARE_BONUS_COUNT);
+	public static Score spare(Score score) {
+		return Score.of(score.value, SPARE_BONUS_COUNT);
 	}
```

Comment:
이전에 코멘트 드렸던 내용이 다소 모호했던 것 같네요 :pray:

```suggestion
	public static Score spare() {
		return Score.of(10, SPARE_BONUS_COUNT);
	}
```

입력으로 받는 score의 value가 항상 10인 것 같아서
꼭 score을 입력으로 받아야 하는지에 대한 질문이였습니다 :)